project.godot

```diff
@@ -97,10 +97,13 @@ locale/translations_pot_files=PackedStringArray("res://scenes/npcs/talker/defaul
 
 2d_physics/layer_1="player"
 2d_physics/layer_2="npcs"
+2d_physics/layer_3="player_detectors"
 2d_physics/layer_5="walls"
 2d_physics/layer_6="interactable"
 
 [rendering]
 
 renderer/rendering_method="gl_compatibility"
 renderer/rendering_method.mobile="gl_compatibility"
+2d/snap/snap_2d_transforms_to_pixel=true
+2d/snap/snap_2d_vertices_to_pixel=true
```

Comment:
What does this part of the change do? To me it seems worth explaining separately, perhaps in its own commit.

Reply:
Totally! It changes the sprites transforms so that pixels get translated as wholes and there is no subpixel movement. It reduces some jitter and wobblyness that you may see otherwise. I'll get it in another commit!